snare.py

```diff
@@ -453,12 +457,20 @@ def parse_timeout(timeout):
     parser.add_argument("--auto-update", help="auto update SNARE if new version available ", default=True)
     parser.add_argument("--update-timeout", help="update snare every timeout ", default='24H')
     parser.add_argument("--server-header", help="set server-header", default='nginx')
+    parser.add_argument("--log_path", help="path to the log file")
     args = parser.parse_args()
     base_path = '/opt/snare/'
     base_page_path = '/opt/snare/pages/'
     config = configparser.ConfigParser()
     config.read(os.path.join(base_path,args.config))
-
+	
+    if args.log_path:
+        log_debug = args.log_path + "snare.log"
```

Comment:
it is not clear that `log_path` is a directory. If I were new for snare, I would put full path to log something like /foo/bar/snare.log